### PR TITLE
Oxxion Rtd Module: add bid filtering

### PR DESCRIPTION
**Pull request by matthieularere-msq**
## Type of change
- [x] Feature

## Description of change
Add a bidder filtering functionnality.

### CODE REVIEW

File: modules/oxxionRtdProvider.md [hunk 1]
```diff
@@ -23,14 +23,16 @@ Then add the oxxion Rtd module to your prebid configuration :
 pbjs.setConfig(
   ...
   realTimeData: {
-    auctionDelay: 200,
+    auctionDelay: 300,
```
Comment (patmmccann) [inline modules/oxxionRtdProvider.md:26]: typo on line 15 above

File: modules/oxxionRtdProvider.js [hunk 2]
```diff
@@ -1,34 +1,71 @@
 import { submodule } from '../src/hook.js'
-import { deepAccess, logInfo } from '../src/utils.js'
+import { deepAccess, logInfo, logError } from '../src/utils.js'
+import MD5 from 'crypto-js/md5.js';
```
Comment (patmmccann) [inline modules/oxxionRtdProvider.js:3]: why not use something supported already?

File: modules/oxxionRtdProvider.md (hunk 1 above)
Comment (matthieularere-msq) [reply modules/oxxionRtdProvider.md:26]: You meant to add the new use case in the description ?

Comment (patmmccann) [reply modules/oxxionRtdProvider.md:26]: you write rtbModule instead of rtdmodule

File: modules/oxxionRtdProvider.js (hunk 2 above)
Comment (patmmccann) [reply modules/oxxionRtdProvider.js:3]: We just don't love the include, isnt sha supported in the browser? I forget. md5 is a rather large build side include I believe.

Comment (patmmccann) [reply modules/oxxionRtdProvider.js:3]: https://github.com/mbaersch/cyrb53-hasher is in python; fairly simple to implement in go just asking chatgpt to translate our js function to go

File: modules/oxxionRtdProvider.md (hunk 1 above)
Comment (matthieularere-msq) [reply modules/oxxionRtdProvider.md:26]: Thanks, I hadn't seen it. It's fixed.

File: modules/oxxionRtdProvider.js (hunk 2 above)
Comment (patmmccann) [reply modules/oxxionRtdProvider.js]: Here's an alternative we use at raptive  https://caniuse.com/?search=digest ; it has wide support for sha-1 and sha-256

Comment (matthieularere-msq) [reply modules/oxxionRtdProvider.js]: just like I said, I removed it for now as it's not needed yet.

**Review by patmmccann — APPROVED** [review]

**Pull request merged**